### PR TITLE
Relax three letter conversion

### PR DESCRIPTION
**Pull request by dnil**
This PR fixes a bug.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
I guess the alternative is explicitly casting each group element as str - or writing a slightly more general conversion function. 😅 This ought to do the trick, but it is possible the end point dbs may support a few more kinds of variations than plain missense, stop or del,

File: scout/constants/__init__.py [hunk 1]
```diff
@@ -173,4 +173,6 @@
     "Trp": "W",
     "Tyr": "Y",
     "Val": "V",
+    "Ter": "*",
```
Comment (northwestwitch) [inline scout/constants/__init__.py:176]: What's Ter, the third letter?

**Review by northwestwitch — APPROVED** [review]
Perhaps it can be directly converted into a hotfix format?

File: scout/constants/__init__.py (hunk 1 above)
Comment (dnil) [reply scout/constants/__init__.py:176]: Stop (Ter-minationcodon)

**Comment by dnil** [comment]
Sure, but we don't really have anything else in master right now, do we? So may as well just patch release in good order.

**Comment by northwestwitch** [comment]
> Sure, but we don't really have anything else in master right now, do we? So may as well just patch release in good order.

like bugs?

**Pull request merged**